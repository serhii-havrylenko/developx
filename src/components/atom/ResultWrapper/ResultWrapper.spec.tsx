import { shallow } from 'enzyme';
import * as React from 'react';

import ResultWrapper from './ResultWrapper';

describe('<ResultWrapper /> tests', () => {
  test('should match snapshot', () => {
    const tree = shallow(<ResultWrapper />);
    expect(tree).toMatchSnapshot();
  });
});
