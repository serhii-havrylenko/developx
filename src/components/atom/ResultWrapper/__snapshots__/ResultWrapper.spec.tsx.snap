// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultWrapper /> tests should match snapshot 1`] = `
.c0 {
  margin-bottom: 20px;
}

<div
  className="c0"
/>
`;
